src/main/java/com/andersen/hw/config/DatabaseConnectionManager.java

```diff
@@ -0,0 +1,16 @@
+package com.andersen.hw.config;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnectionManager {
+
+    private static final String URL = "jdbc:postgresql://localhost:5433/my_ticket_service_db";
+    private static final String USER = "root";
+    private static final String PASSWORD = "password";
```

Comment:
good practice to move this values in some property file in resources

Reply:
done